src/config/server.rs

```diff
@@ -47,6 +49,12 @@ impl Server {
   pub fn get_global_response_timeout(&self) -> i64 {
     self.global_response_timeout.unwrap_or(0)
   }
+
+  pub fn get_http_options(&self) -> HttpOptions {
+    let default_http_options = HttpOptions { version: HttpVersion::HTTP1, cert_path: None, key_path: None };
+    self.http.clone().unwrap_or(default_http_options)
```

Comment:
Can we use `unwrap_or_default` instead of manually creating the default value?
```suggestion
    self.http.clone().unwrap_or_default()
```